docs/mobile-apps/real-device-cleaning.md

```diff
@@ -0,0 +1,59 @@
+---
+id: real-device-cleaning
+title: Real Device Cleaning Process
+sidebar_label: Real Device Cleaning Process
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The cleaning process runs at the end of each test session for Sauce Labs Real Device. It resets the device back to a default state, ready for the next test. The cleaning process also removes any private data which may have been part of the prior test. If we encounter any issue during the cleaning process, or we fail to clean the device, we put those devices aside for manual inspection to prevent data and security leaks. 
+
+The cleaning process differs slightly between the Public Devices which are shared among all our customers, and the Private Devices, where the owner has more control over the configuration of their device. It also differs between Android and iOS, due to the different features of these operating systems.
+
+## Public Devices
+
+Every 5th cleaning session includes a device reboot. This ensures that the system remains 
+performant, and any stray processes are stopped.
+
+:::note
+A factory reset is not performed between test sessions, as this requires manual intervention and would slow down the time required between test runs.
```

Comment:
This sentence is still a little hard to understand. I think it was meant to say that a factory reset would make the time between test runs take longer. 
```suggestion
A factory reset is not performed between test sessions, as this requires manual intervention and would increase the time required between tests.
```